Mastodon/Supporting Files/AppDelegate.swift

```diff
@@ -65,11 +65,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 extension AppDelegate {
     func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
-        #if DEBUG
-        return .all
```

Comment:
fwiw this code did make it easy to identify and fix issues in landscape mode and may be helpful to keep.

Reply:
Thank you very much, I see your point and you're right. Thing is: Support for landscape on iPhone isn't on the agenda for now and I prefer to have the same code running in both `DEBUG` and `RELEASE`. Does this make sense?

Also change is just on `git revert 9ad565d3236` away 🙂

Reply:
Haha I think I'm just getting emails on this PR because I left those comments before 😅

Makes sense to keep the build time code paths predictable! Maybe we can add it as a runtime option somehow? I was admittedly surprised when landscape worked in the simulator but not in prod using the build time check, so a runtime debug setting might make that clearer / opt-in.